### ISSUE TITLE
Documentation alignment and install

### ISSUE THREAD

**Issue by HaleTom** [robert7/nixnote2#32]
`docs/shortcuts-howto.md`:

> You should have a shortcuts_sample.txt file.  Copy this to shortcuts.txt 
in your Nixnote home directory (~/.nixnote on Linux & OS-X and your user

The package I'm working on has:

```
  install -Dm644 shortcuts.txt "${pkgdir}"/usr/share/doc/nixnote2/shortcuts_sample.txt
```

Note the rename is necessary to match the doco.

Would you consider renaming the file to match the documentation?

Would you also consider putting in `/usr/share/doc/nixnote2`:
* CHANEGLOG.md
* shortcuts-howto.md
* shortcuts_sample.txt (see above)

The following files were in the old Nixnote2 package, in `help/`
* UserDocumentation.odt
* UserDocumentation.pdf

What happened to them?

**Comment by robert7** [comment]
Basically I see all the docs included in the repository as obsolete/deprecated. The "new documentation" is the project wiki. I already created a lot of content there, but did not yet had enough time to do it all. Its improving incrementally. 

For example articles for "how to modify shortcuts" & "how to modify themes" are not yet created.

Part of the problem is that you are looking at the tag "2.1.0-beta3" but there already have been updates since "beta3".

I created a new tag for you - "v2.1.0-beta4g" which is more up to date, then "beta3". Pls use this for your tests. 

1. CHANGELOG.md is now just a "redirect" to "debian/changelog". I would delete it altogether, but I linked to it in some issues ... 
2. shortcuts-howto.txt ... I updated it a bit, but it is about to be moved to wiki and updated. The "shortcuts_sample.txt" is not needed anymore. Its better to use the provided "shortcuts.txt" which is part of the app and this will be explained in the wiki article (detto it works for themes.txt)
3. User-Documentantion - is partialy obsolete - the new documentation is in the wiki - but the original text from v2.0 is 1:1 is already copied in the wiki - its here https://github.com/robert7/nixnote2/wiki/Legacy-Documentation - I plan later to annotate it where its obsolete etc.

So basically my idea of docs deployed the the application in /usr/share/doc/nixnote2 is:
1. changelog, copyright, licence
2. and one readme.txt containing something like "for docs pls look in the wiki: https://github.com/robert7/nixnote2/wiki"

My plan for next steps are:
1, bring out stable release 2.0.1
2. improve the docs in wiki

The wiki is also open to contribution from other people, if anyone is interested.

**Comment by HaleTom** [comment]
Sweet, thanks for explaining.

I'm building `v2.1.0-beta4g` as I type.

Having a top-level changelog (even if symlink) is probably a Good Thing - not everyone is a debian-ite :)

**Comment by HaleTom** [comment]
Reopening: you may have been using this as a tracker for the `shortcuts.txt`. 

Happy to have this closed as you wish.

**Comment by HaleTom** [comment]
I just went to check out the wiki... 

I couldn't work out how to contribute to it (I'm planning to add my forthcoming package).

May I suggest a "Contributing" heading with how-to on the front page?

**Comment by robert7** [comment]
>Reopening: you may have been using this as a tracker for the shortcuts.txt.

OK

>Having a top-level changelog (even if symlink) is probably a Good Thing

OK, I added a symlink

>May I suggest a "Contributing" heading

I added "CONTRIBUTING.md" with some stub content for now.

the changes are in "feature/rc1" - I'll merge later in develop

**Comment by robert7** [comment]
>you may have been using this as a tracker for the shortcuts.txt.

content of docs/shortcuts_sample.txt moved to: https://github.com/robert7/nixnote2/wiki/HowTo---Customize-keyboard-shortcuts

>I couldn't work out how to contribute to it (I'm planning to add my forthcoming package).

The wiki "edit" should now be accessible.

I think this issue can be now closed.

**Comment by HaleTom** [comment]
Cheers! I just added a page on changing the default notebook :)

**Comment by robert7** [comment]
ok thanks!

I just did the same before noticing your comment.. so I joined the 2 pages
now its on https://github.com/robert7/nixnote2/wiki/HowTo---Set-default-notebook-for-new-notes
